godotenv.go

```diff
@@ -124,6 +115,35 @@ func Parse(r io.Reader) (envMap map[string]string, err error) {
 	return
 }
 
+// ReadAllLines will convert the Reader into an array of strings
+// one for each line in the original text.
+func ReadAllLines(r io.Reader) (lines []string, err error) {
+	bf := bufio.NewReader(r)
+	linePart, isPrefix, err := bf.ReadLine()
+
+	for err == nil {
+		lineStr := string(linePart)
+
+		// handling of lines larger than the default buffer size
+		for err == nil && isPrefix == true {
+			linePart, isPrefix, err = bf.ReadLine()
+			if err != nil {
+				break
+			}
+			lineStr += string(linePart)
+		}
+
+		// `err == nil` is used to skip lines which caused an error
+		if isPrefix == false && err == nil {
```

Comment:
This condition should supposedly allow skipping "invalid" lines.
But I am not sure if such a situation could even exist...
Perhaps I should just `return` immediately if/when an err is spotted?